Sources/ArcGISToolkit/Components/Compass/Compass.swift

```diff
@@ -110,11 +109,15 @@ public struct Compass: View {
                                  """
                     )
                 )
+#if os(visionOS)
+                .hoverEffect { effect, isActive, _ in
+                    effect.scaleEffect(isActive ? 1.05 : 1.0)
+                }
+#endif
```

Comment:
This is our first time using a custom hover effect so let me know your thoughts. The original hover effect I found was not even noticeable on a device (it was a little better on a simulator). This hover effect will increase the size of the compass very slightly when you look at it to indicate you can tap it. Apple has used something similar to this in some places. Here is what it looks like.
![Simulator Screen Recording - Apple Vision Pro - 2025-01-09 at 11 56 31](https://github.com/user-attachments/assets/3639d803-c857-424c-a510-c4268d3fbf0e)

Reply:
@philium Please test this on your device too.

Reply:
I did test it and it was barely visible to me, but perhaps other can see it better.

Reply:
Do you think the scale should be larger when the hover effect is active? I'm fine with how it is now but increasing it a bit couldn't hurt. Just as long as it isn't too big where it becomes distracting.

Reply:
It's up to you. Probably more significant would be a highlight while the compass is clicked/tapped, before the action is actually triggered.

Reply:
>  before the action is actually triggered.

That sounds like what the original hover effect was like and it was hard to notice in my testing.

Reply:
Could we highlight and scale?

Reply:
I bet we can, I can see what that looks like.

Reply:
I think it looks good:
![Simulator Screen Recording - Apple Vision Pro - 2025-01-09 at 14 44 12](https://github.com/user-attachments/assets/f709edd7-7623-4f14-83ee-6652f10e9960)